deploy/contents/install/app/install-utils.sh

```diff
@@ -627,6 +632,10 @@ function parse_options {
     esac
     done
     set -- "${POSITIONAL[@]}" # restore positional parameters
+
+    if [ -z "${CP_SERVICES_ENABLED}" ]; then
```

Comment:
we should make `CP_SERVICES_ENABLED=all` only in case when based on our logic `CP_INSTALL_SERVICES_ALL == 1`. Based on code this place is here:  https://github.com/epam/cloud-pipeline/pull/3441/files#diff-5c368da199a8aa904a21120fa49deab5adb5dfe6f1f66389a61ec978407c7855L745-L748

So, basically we should define `CP_SERVICES_ENABLED="all"` in the same block of code.

Reply:
Fixed